src/main/java/gregtech/loaders/MaterialInfoLoader.java

```diff
@@ -82,6 +83,56 @@ public static void init() {
+                new MaterialStack(Materials.Copper, OrePrefix.cableGtSingle.materialAmount * 2),
+                new MaterialStack(Materials.Rubber, OrePrefix.plate.materialAmount * 2)));
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[3].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(Materials.StainlessSteel, OrePrefix.plate.materialAmount * 8),
+                new MaterialStack(Materials.Gold, OrePrefix.cableGtSingle.materialAmount * 2),
+                new MaterialStack(Materials.Rubber, OrePrefix.plate.materialAmount * 2)));
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[4].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(Materials.Titanium, OrePrefix.plate.materialAmount * 8),
+                new MaterialStack(Materials.Aluminium, OrePrefix.cableGtSingle.materialAmount * 2),
+                new MaterialStack(Materials.Rubber, OrePrefix.plate.materialAmount * 2)));
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[5].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(Materials.TungstenSteel, OrePrefix.plate.materialAmount * 8),
+                new MaterialStack(Materials.Tungsten, OrePrefix.cableGtSingle.materialAmount * 2),
+                new MaterialStack(Materials.Rubber, OrePrefix.plate.materialAmount * 2)));
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[6].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(Materials.RhodiumPlatedPalladium, OrePrefix.plate.materialAmount * 8),
+                new MaterialStack(Materials.VanadiumGallium, OrePrefix.cableGtSingle.materialAmount * 2),
+                new MaterialStack(Materials.Rubber, OrePrefix.plate.materialAmount * 2)));
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[7].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(Materials.NaquadahAlloy, OrePrefix.plate.materialAmount * 8),
+                new MaterialStack(Materials.Naquadah, OrePrefix.cableGtSingle.materialAmount * 2),
+                new MaterialStack(Materials.Rubber, OrePrefix.plate.materialAmount * 2)));
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[8].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(Materials.Livermorium, OrePrefix.plate.materialAmount * 8),
+                new MaterialStack(Materials.NaquadahAlloy, OrePrefix.cableGtSingle.materialAmount * 2),
+                new MaterialStack(Materials.Rubber, OrePrefix.plate.materialAmount * 2)));
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[14].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(Materials.Neutronium, OrePrefix.plate.materialAmount * 8),
+                new MaterialStack(Materials.RutheniumTriniumAmericiumNeutronate, OrePrefix.cableGtSingle.materialAmount * 2),
+                new MaterialStack(Materials.Rubber, OrePrefix.plate.materialAmount * 2)));
+
```

Comment:
These had to be done separately since the handcrafting recipe is more resource-expensive than the Assembler recipe, leading to potential dupes. If we add a method onto `RecipeBuilder` to calculate unification data, these can be removed (as can the entries for EBF coils)

Reply:
Minor nitpick that should not really matter. These should use the proper assembler fluid materials, instead of all just using rubber